packages/foreman/nodejs-history/nodejs-history.spec

```diff
@@ -16,38 +19,49 @@ Source5: https://registry.npmjs.org/value-equal/-/value-equal-0.4.0.tgz
 Source6: https://registry.npmjs.org/resolve-pathname/-/resolve-pathname-2.2.0.tgz
 Source7: https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.13.2.tgz
 Source8: https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz
-Source9: %{name}-%{version}-registry.npmjs.org.tgz
-BuildRequires: nodejs-packaging
+Source9: nodejs-%{npm_name}-%{version}-registry.npmjs.org.tgz
+
+Requires:       rh-nodejs10-nodejs
+BuildRequires:  rh-nodejs10-nodejs-devel
+BuildRequires:  rh-nodejs10-build
+BuildRequires:  rh-nodejs10-npm
+
 BuildArch: noarch
 ExclusiveArch: %{nodejs_arches} noarch
 
-Provides: npm(%{npm_name}) = %{version}
-Provides: bundled(npm(@babel/runtime)) = 7.4.3
-Provides: bundled(npm(history)) = 4.9.0
-Provides: bundled(npm(js-tokens)) = 4.0.0
-Provides: bundled(npm(loose-envify)) = 1.4.0
-Provides: bundled(npm(regenerator-runtime)) = 0.13.2
-Provides: bundled(npm(resolve-pathname)) = 2.2.0
-Provides: bundled(npm(tiny-invariant)) = 1.0.4
-Provides: bundled(npm(tiny-warning)) = 1.0.2
-Provides: bundled(npm(value-equal)) = 0.4.0
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
+Provides: bundled(%{?scl_prefix}npm(@babel/runtime)) = 7.4.3
+Provides: bundled(%{?scl_prefix}npm(history)) = 4.9.0
+Provides: bundled(%{?scl_prefix}npm(js-tokens)) = 4.0.0
+Provides: bundled(%{?scl_prefix}npm(loose-envify)) = 1.4.0
+Provides: bundled(%{?scl_prefix}npm(regenerator-runtime)) = 0.13.2
+Provides: bundled(%{?scl_prefix}npm(resolve-pathname)) = 2.2.0
+Provides: bundled(%{?scl_prefix}npm(tiny-invariant)) = 1.0.4
+Provides: bundled(%{?scl_prefix}npm(tiny-warning)) = 1.0.2
+Provides: bundled(%{?scl_prefix}npm(value-equal)) = 0.4.0
```

Comment:
Is this the correct way to specify bundled with an scl_prefix ?

Reply:
I think so